pkg/mount/mount.go

```diff
@@ -38,9 +40,10 @@ func Mount(device, target, mType, options string) error {
 	return ForceMount(device, target, mType, options)
 }
 
-// Mount the specified options at the target path
-// reguardless if the target is mounted or not
-// Options must be specified as fstab style
+// Mount a filesystem according to the specified configuration, regardless if
```

Comment:
@LK4D4 AFAIK, `Mount` will check first to see if a similar mount exists. If it does, an error is returned; if it nothing exists, it calls `ForceMount`.